package.json

```diff
@@ -11,15 +11,14 @@
     "copy-sjcl": "node scripts/generateSJCL.js",
     "bundle-config": "node scripts/bundleConfig.mjs",
     "build": "npm run build-locales && npm run build-firefox && npm run build-chrome && npm run build-apple && npm run build-android && npm run build-windows && npm run build-integration && npm run build-chrome-mv3",
-    "build-locales": "mkdir -p build/locales && node scripts/buildLocales src/locales/click-to-load > build/locales/ctl-locales.js",
-    "build-firefox": "mkdir -p build/firefox/ && node scripts/inject.js firefox > build/firefox/inject.js",
-    "build-chrome": "mkdir -p build/chrome/ && node scripts/inject.js chrome > build/chrome/inject.js",
-    "build-chrome-mv3": "mkdir -p build/chrome-mv3/ && node scripts/inject.js chrome-mv3 > build/chrome-mv3/inject.js",
-    "build-apple": "mkdir -p Sources/ContentScopeScripts/dist/ && node scripts/inject.js apple > Sources/ContentScopeScripts/dist/contentScope.js",
-    "build-android": "mkdir -p build/android/ && node scripts/inject.js android > build/android/contentScope.js",
-    "build-windows": "mkdir -p build/windows/ && node scripts/inject.js windows > build/windows/contentScope.js",
-    "build-integration": "mkdir -p build/integration/ && node scripts/inject.js integration > build/integration/contentScope.js && npm run copy-build-integration",
-    "copy-build-integration": "cp build/integration/contentScope.js integration-test/extension",
+    "build-locales": "node scripts/buildLocales.js --dir src/locales/click-to-load --output build/locales/ctl-locales.js",
+    "build-firefox": "node scripts/inject.js --entry inject/mozilla.js --output build/firefox/inject.js --platform firefox",
+    "build-chrome": "node scripts/inject.js --entry inject/chrome.js --output build/chrome/inject.js --platform chrome",
+    "build-chrome-mv3": "node scripts/inject.js --entry inject/chrome-mv3.js --output build/chrome-mv3/inject.js --platform chrome-mv3",
+    "build-apple": "node scripts/inject.js --entry inject/apple.js --output Sources/ContentScopeScripts/dist/contentScope.js --platform apple",
+    "build-android": "node scripts/inject.js --entry inject/android.js --output build/android/contentScope.js --platform android",
+    "build-windows": "node scripts/inject.js --entry inject/windows.js --output build/windows/contentScope.js --platform windows",
+    "build-integration": "node scripts/inject.js --entry inject/integration.js --output build/integration/contentScope.js --output integration-test/extension/contentScope.js --platform integration",
```

Comment:
The script supports multiple outputs, so no need for a separate 'copy' script 💪🏻

Reply:
Nice we needed this change for sourcemaps anyway.